default.conf.template

```diff
@@ -5,45 +5,67 @@
 
 # The docker cli uses the realm domain given in the www-authenticate to auth.
 # So we need to re-write from ghcr.io to our proxy.
+# Use a map as it is built in, even though its not a clean way to replace strings.
+# Ideally would use more-headers module, but its not in the default image.
 map $upstream_http_www_authenticate $auth_header {
         # used to have 1 occurance of domain, now has 2. Handle both.
         ~(.*)ghcr.io(.*)ghcr.io(.*) $1${PROXY_DOMAIN}$2${PROXY_DOMAIN}$3;
         ~(.*)ghcr.io(.*)            $1${PROXY_DOMAIN}$2;
 }
 
+proxy_http_version 1.1; # keep alives to gchr.io
+proxy_buffering off;    # start streaming response immeadiately
+#error_log /var/log/nginx/error-debug.log debug;
+
 server {
 
-    # be idendified by either domain
+    # be identified by either domain
     server_name ${PROXY_DOMAIN} ${PROXY_ORIGIN};
     root /var/www/html;
     listen ${PORT};
 
     # we need a resolver configured so we can look up domains when redirecting
     resolver ${RESOLVER};
 
-    location / {
-        # Only GET/HEAD
+    # excat match, initial query, rewrite www-authentication header
+    location = /v2/ {
```

Comment:
does this mean that `limit_except` only applies to exactly `/v2/`, and not to anything else (e.g. `/v2/opensafely/blah/`)?

Reply:
the limit_except clause is repeated for each explicit location, so it does apply for /v2/opensafely/blah, however I think you have a point that it doesn't apply to /v2/other-org, which I'll add.